web/index.html

```diff
@@ -195,6 +195,15 @@ <h3 class="header2 light"><small class="header">Can select multiple options</sma
                     </label>
                   </div>
                 </div>
+                <div class="col-md-12">
+                  <div class="col s12" style="position: relative;">
+                    <h3 class="header light">Are you looking for something else?</h3>
```

Comment:
```suggestion
                    <h3 class="header light">Are you looking for some other resource?</h3>
```

Since we want to stick to the only resource for this section. Otherwise, user can also comment on the city which can create false search results.

Reply:
Done